### PR TITLE
Demo purposes/merge andreea mircea

### PR DESCRIPTION
**Pull request by AndreeaDora**

### CODE REVIEW

File: docker-compose.consul.prod.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+rethinkdb:
+    restart: always
+    image: rethinkdb
+    ports:
+        - "8080"
+        - "28015"
+        - "29015"
+#    environment:
```
Comment (rqc) [inline docker-compose.consul.prod.yml]: @AndreeaDora @borzamircea please cleanup all of these commented proxy statements.

File: docker-compose.external.swarm.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    image: rethinkdb
+    ports:
+        - "8080"
+        - "28015"
+        - "29015"
+
+controller:
+    restart: always
+    image: shipyard/shipyard:latest
+    command: server -d tcp://myd-vm23183.hpswlabs.adapps.hp.com:3375
```
Comment (rqc) [inline docker-compose.external.swarm.yml]: @AndreeaDora @borzamircea please try to make the swarm url to be generic. One thing that you can attempt is to set an env var in your shell with the url, then inject it to the container through compose and then reference it in the command line. Here is an example:

```
controller:
    ...
    command: server -d $SHIPYARD_SWARM_ENDPOINT 
    ...
    # note the empty value for the SHIPYARD_SWARM_ENDPOINT key below
    environment:
        SHIPYARD_SWARM_ENDPOINT:
```

Then on your shell environment

```
export SHIPYARD_SWARM_ENDPOINT="tcp://myd-vm23183.hpswlabs.adapps.hp.com:3375"
docker-compose -f docker-compose.external.swarm.yml up
```

**Comment by rqc** [comment]
Guys, it is looking good. I think we could tweak it to make it more generic and clean it up a bit. Even if there is no chance to do those changes, we can merge it to the `dev` branch and then fix those things after the demo.

Thanks!

**Comment by soninob** [comment]
@AndreeaDora, @borzamircea  - I agree with the above - I suggest you will take care of the case when the env var is not supplied. A proper message should be displayed.

**Pull request merged**